### PR TITLE
6927: updated button for upload editor

### PR DESCRIPTION
**Pull request by nikola-gladovic**
Signed-off-by: Nikola <nikola.gladovic@nextcloud.com>

#6927

### CODE REVIEW

File: src/components/UploadEditor.vue [hunk 1]
```diff
@@ -45,14 +45,16 @@
 					</template>
 					<div :key="'addMore'"
 						class="add-more">
-						<button :aria-label="addMoreAriaLabel"
-							class="add-more__button primary"
+						<Button :aria-label="addMoreAriaLabel"
```
Comment (marcoambrosini) [inline src/components/UploadEditor.vue:48]: I don't think we want to change this one @nikola-gladovic. The original is much bigger than 44px right? Could you please add screenshots of how the button looked before the swap and after on all of your open button pull requests?

Comment (nikola-gladovic) [reply src/components/UploadEditor.vue:48]: Ok, I will do that. @marcoambrosini  :)

Comment (marcoambrosini) [reply src/components/UploadEditor.vue:48]: Thanks!

**Comment by nikola-gladovic** [comment]
image

**Review by marcoambrosini — APPROVED** [review]

**Pull request merged**